### PR TITLE
from __future__ import print_function

### PR DESCRIPTION
**Pull request by cclauss**
__print()__ is a function in modern Python

### CODE REVIEW

**Comment by BoldMonday** [comment]
I appreciate the suggestion, but the environment in which this Python script is run (FontLab Studio 5), together with the required RoboFab library, is Python 2.7. That will not change.

Future build scripts might move to other software and other environments.

**Comment by cclauss** [comment]
That is OK.  This code is compatible with both Python 2 and Python 3.

File: scripts/IBM Plex export FDK files.py [hunk 1]
```diff
@@ -362,11 +359,8 @@ def checkCurveType(f):
 				if hasType == None:
 					typeDict[pType] = True
 
-	allTypes = typeDict.keys()
-	if 'curve' in allTypes:
-		return True
-	else:
-		return False
+	allTypes = typeDict
+	return bool('curve' in allTypes)
```
Comment (teners) [inline scripts/IBM Plex export FDK files.py]: You don't really need `bool()` here, the expression returns bool itself :)

Comment (cclauss) [reply scripts/IBM Plex export FDK files.py]: "Explicit is better than implicit."

Comment (cclauss) [reply scripts/IBM Plex export FDK files.py]: Removed the __bool()__

**Pull request closed without merging**